recipe/bld.bat

```diff
@@ -170,46 +173,52 @@ if errorlevel 1 exit /b 1
 if "%PKG_NAME%" == "libtorch" (
     @REM Extract the compiled wheel into a temporary directory
     if not exist "%SRC_DIR%/dist" mkdir %SRC_DIR%/dist
-    pushd %SRC_DIR%/dist
+    pushd %SRC_DIR%/dist || goto :error
     for %%f in (../torch-*.whl) do (
-        wheel unpack %%f
+        wheel unpack %%f || goto :error
     )
 
     @REM Navigate into the unpacked wheel
-    pushd torch-*
+    pushd torch-* || goto :error
 
     @REM Move the binaries into the packages site-package directory
-    robocopy /NP /NFL /NDL /NJH /E torch\bin %SP_DIR%\torch\bin\
-    robocopy /NP /NFL /NDL /NJH /E torch\lib %SP_DIR%\torch\lib\
-    robocopy /NP /NFL /NDL /NJH /E torch\share %SP_DIR%\torch\share\
+    robocopy /NP /NFL /NDL /NJH /E torch\bin %SP_DIR%\torch\bin\ || goto :error
+    robocopy /NP /NFL /NDL /NJH /E torch\lib %SP_DIR%\torch\lib\ || goto :error
+    robocopy /NP /NFL /NDL /NJH /E torch\share %SP_DIR%\torch\share\ || goto :error
```

Comment:
You need to be careful with robocopy, which has an exit code of 1 if files were successfully copied (I [learned](https://github.com/conda-forge/clangdev-feedstock/blob/d5a5e4aff22133083f5c8ab87a100a2652f3d935/recipe/bld.bat#L7-L8) the hard way too).